src/js/index.js

```diff
@@ -1,12 +1,25 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
-var makeSubmitForm = require('./makeSubmitForm.jsx');
+var api = require('./api');
+var userSelect = require('./userSelect.jsx');
+var fiContainer = require('./fiContainer.jsx');
 
-var SubmitForm = makeSubmitForm('/submit', function(e){
-  console.log(e, 'file uploaded');
-});
 
 ReactDOM.render(
```

Comment:
Sorry @wpears. That comment early was for this `userSelect`. We won't need it later, or in production, right?